app/views/internal/response_templates/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_for [:internal, response_template] do |f| %>
+  <%= f.label :type_of, "Type of Response Template (Mod Comment, Personal Comment, Email, etc)" %>
+  <br>
+  <% type_of_options = [["Mod Comment", "mod_comment"], ["Personal Comment", "personal_comment"], ["Email", "email_reply"], ["Abuse Report Email", "abuse_report_reply"]] %>
+  <%= f.select :type_of, options_for_select(type_of_options, response_template.type_of), html: { required: true } %>
+  <br>
+  <%= f.label :title %>
+  <%= f.text_field :title, value: response_template.title %>
```

Comment:
Instead of using `<br>`s can we use the Bootstrap form styles like we do elsewhere?

For example, something like:
```
<div class="form-group">
  <%= f.label :title %>
  <%= f.text_field :title, value: response_template.title, class: "form-control" %>
</div>
...
<%= f.submit "Submit", class: "btn btn-primary my-2" %>
<% end %>
```